tests/sync/genesis/test_genesis_voting.py

```diff
@@ -0,0 +1,79 @@
+import time
+import pytest
+from pytest_testconfig import config as testconfig
+
+import tests.queries as q
+from tests.sync.test_sync import new_client_in_namespace
+from tests.test_bs import get_conf, setup_bootstrap, start_poet, add_curl
+
+# ==============================================================================
+#    TESTS
+# ==============================================================================
+
+
+# set a X nodes cluster
+# sleep in order to enable a block to be delivered
+# validate a block has been delivered
+# add an additional client while still in genesis
+# sleep until layer 4
+# validate no errors occurred
+# validate new node didn't receive any new blocks before being synced
+@pytest.mark.xfail(strict=True, reason="new pod creates blocks before being synced")
+def test_unsync_while_genesis(init_session, setup_bootstrap, start_poet, add_curl):
+    time_before_first_block = 70
+    layers_to_wait = 4
+
+    layer_duration = int(testconfig['client']['args']['layer-duration-sec'])
+    bs_info = setup_bootstrap.pods[0]
+    cspec = get_conf(bs_info, testconfig['client'], None, setup_bootstrap.pods[0]['pod_ip'])
+
+    # Create a cluster of nodes
+    _ = new_client_in_namespace(testconfig['namespace'], setup_bootstrap, cspec, 9)
```

Comment:
does this mean a client has been created when this function is returned?

Reply:
Yes, in the future I will merge this function with the test_bs creation func